lib/srv/desktop/rdp/rdpclient/Cargo.toml

```diff
@@ -21,5 +21,5 @@ num-traits = "0.2"
 # RustCrypto doesn't expose the low-level primitives we need for the smartcard
 # challenge signing (see src/piv.rs for details).
 openssl = "0.10.36"
-rdp-rs = { git = "https://github.com/gravitational/rdp-rs", rev = "92a80a25e27a802ffb2d21ac9d628e27e831dd5c" }
+rdp-rs = { git = "https://github.com/gravitational/rdp-rs", branch = "andrew/mouse-scroll" }
```

Comment:
TODO(me): change this to a master `rev` after https://github.com/gravitational/rdp-rs/pull/3 gets merged